Cover.xs

```diff
@@ -4,7 +4,7 @@
  * This software is free.  It is licensed under the same terms as Perl itself.
  *
  * The latest version of this software should be available from my homepage:
- * http://www.pjcj.net
+ * https://pjcj.net
```

Comment:
both domains were used in this repository. Everything redirects to the shorter domain w/ https, so unless there's a specific reason to use http: or the longer www, let's use the one following the redirect.